pkg/services/ngalert/api/tooling/cmd/clean-swagger/main.go

```diff
@@ -63,6 +63,39 @@ func main() {
 		}
 	}
 
+	paths, ok := data["paths"].(map[string]interface{})
+	if !ok {
+		log.Fatal("no paths")
+	}
+
+	for _, path := range paths {
+		path, ok := path.(map[string]interface{})
+		if !ok {
+			log.Fatal("invalid path")
+		}
+
+		for _, op := range path {
+			op, ok := op.(map[string]interface{})
+			if !ok {
+				continue
+			}
+
+			tags, ok := op["tags"].([]interface{})
+			if !ok {
+				log.Println("invalid op, skipping")
+				continue
+			}
+
+			// Remove "stable" tag. Multiple tags cause routes to render strangely in the final docs.
```

Comment:
Hide the stable tag. It's just used for knowing which doc to send the route to. We don't want to codegen any extra go files based on it. Also, the swagger UI looks super weird when a route has multiple tags - the route shows up many times in the doc.